packages/frontend/src/pages/Landing/components/AppIntegration.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Flex, Image, Text } from '@chakra-ui/react'
+import { APP_ICON_URL } from 'config/urls'
+
+export interface AppIntegrationProps {
+  iconName: string
+  name: string
+  description: string
+}
+
+export default function AppIntegration(props: AppIntegrationProps) {
+  const { iconName, name, description } = props
+  return (
+    <Flex
+      flexDir="column"
+      borderRadius="md"
+      p={4}
+      bg="secondary.50"
+      gap={4}
+      h="100%"
+    >
+      <Image
+        src={APP_ICON_URL(iconName)}
```

Comment:
nit: technically not a constant although i know what you're getting at...

```suggestion
        src={appIconUrl(iconName)}
```

Reply:
discussed to keep it caps based on the file